liquidctl/driver/icue_rgb_elite.py

```diff
@@ -0,0 +1,408 @@
+"""liquidctl drivers for Corsair iCUE RGB ELITE
```

Comment:
Please integrate your changes with the existing driver. There's no need for this much duplication.

In fact, so far the only functional change your code does on top of the `hydro_platinum` module/driver is:

```patch
+        (0x1b1c, 0x0c35, 'Corsair iCUE H100i RGB ELITE',
+            {'fan_count': 2, 'fan_leds': 0, 'has_pump':True, 'led_channels':1}),
```

And note that `'led_channels': 1` currently does nothing.

Reply:
Moved to `hydro_platinum` in [ last commit #557](https://github.com/liquidctl/liquidctl/pull/557/commits/ae478d5adbbd96376c712538e45f77cd7295f8e4)

Tested without `led_channels:1`, the pump led control still works as expected : )